kube_helpers.go

```diff
@@ -413,17 +359,17 @@ func createRole(namespace string, role string, roleTemplateFile string) (bool, s
 // labelResource applies Helmsman specific labels to Helm's state resources (secrets/configmaps)
 func labelResource(r *release) {
 	if r.Enabled {
-		log.Println("INFO: applying Helmsman labels to [ " + r.Name + " ] in namespace [ " + r.Namespace + " ] ")
-		storageBackend := "configmap"
+		logs.Info("applying Helmsman labels to [ " + r.Name + " ] in namespace [ " + r.Namespace + " ] ")
+		storageBackend := "secret"
 
-		if s.Settings.StorageBackend == "secret" {
-			storageBackend = "secret"
+		if s.Settings.StorageBackend != "" {
+			storageBackend = s.Settings.StorageBackend
 		}
 
 		cmd := command{
 			Cmd:         "kubectl",
-			Args:        []string{"label", storageBackend, "-n", getDesiredTillerNamespace(r), "-l", "NAME=" + r.Name, "MANAGED-BY=HELMSMAN", "NAMESPACE=" + r.Namespace, "TILLER_NAMESPACE=" + getDesiredTillerNamespace(r), "--overwrite"},
-			Description: "applying labels to Helm state in [ " + getDesiredTillerNamespace(r) + " ] for " + r.Name,
+			Args:        []string{"label", storageBackend, "-l", "NAME=" + r.Name, "MANAGED-BY=HELMSMAN", "NAMESPACE=" + r.Namespace, "--overwrite"},
```

Comment:
You should run this command in the namespace of the release. So it's better to add `"-n", r.Namespace,` after `storageBackend,`.

Reply:
thanks :-)